### ISSUE TITLE
Add support for OSRS

### ISSUE THREAD

**Issue by mswdev** [michaelbull/rs-api#6]
Add support for OSRS possibly? 

@michaelbull If this is something you don't have time for, I'd be willing to add support for it and make a pull request.

**Comment by michaelbull** [comment]
Do [these](https://github.com/michaelbull/rs-api/blob/master/src/main/java/com/github/michaelbull/rs/hiscores/HiscoreTable.java#L35) not work?

**Comment by mswdev** [comment]
> Do [these](https://github.com/michaelbull/rs-api/blob/master/src/main/java/com/github/michaelbull/rs/hiscores/HiscoreTable.java#L35) not work?

I didn't realize that OSRS was supported in terms of hiscores as the readme and project information only mentions RuneScape generically rather than RuneScape 3 and or OSRS.

What about OSRS G.E data, If I'm not mistaken, that doesn't look to be supported?

**Comment by michaelbull** [comment]
Yeah doesn't look like the OSRS GE is supported, however there is documentation for it [here](https://www.reddit.com/r/2007scape/comments/3g06rq/guide_using_the_old_school_ge_page_api/).

I think it would be a case of tackling [this](https://github.com/michaelbull/rs-api/blob/master/src/main/java/com/github/michaelbull/rs/ge/GrandExchange.java#L21) hardcoded URL, making it an enum along the lines of the [`HiscoresTable`](https://github.com/michaelbull/rs-api/blob/master/src/main/java/com/github/michaelbull/rs/hiscores/HiscoreTable.java#L12), passing it to the functions inside of the `GrandExchange` class - in a similar fashion to the methods inside [`Hiscores`](https://github.com/michaelbull/rs-api/blob/master/src/main/java/com/github/michaelbull/rs/hiscores/Hiscores.java#L206).
